src/query/query_manager.c

```diff
@@ -2841,7 +2785,7 @@ qmgr_free_temp_file_list (THREAD_ENTRY * thread_p, QMGR_TEMP_FILE * tfile_vfid_p
       fd_ret = NO_ERROR;
       if ((tfile_vfid_p->temp_file_type != FILE_QUERY_AREA || is_error) && !VFID_ISNULL (&tfile_vfid_p->temp_vfid))
 	{
-	  if (was_preserved)
```

Comment:
what was the context of was_preserved?
A `preserved` variable in `QMGR_TEMP_FILE` is the same meaning?

Reply:
same meaning because that the QMGR's preserved flag is only set at 'create result file'

Reply:
The result file is whether preserved or not was managed in `query entry`. But now, each result file sets preserved itself. and `query_entry->is_preserved` seems not used anymore. In terms of this changing of context, Is it safe? and `query_entry->is_preserved` is still needed?

Reply:
We discussed that `qmgr_query_entry.is_preserved` becomes redundant variable now. It would better to remove it.